pkgs/by-name/op/openorienteering-mapper/package.nix

```diff
@@ -31,6 +31,11 @@ stdenv.mkDerivation rec {
       url = "https://github.com/OpenOrienteering/mapper/commit/bc52aa567e90a58d6963b44d5ae1909f3f841508.patch";
       hash = "sha256-pQzw2EfVay+0GOtIbh1KJOkILcj5LRMzmMYy9q+abK4=";
     })
+    # https://github.com/OpenOrienteering/mapper/issues/2334
+    (fetchpatch {
+      url = "https://github.com/dg0yt/mapper/commit/5e8bebcc7860ce0b2c9802591a806b294761c0aa.patch";
```

Comment:
Premature!
Please don't discourage me from pushing to my own repo.

Reply:
(FTR that branch this commit belongs to was already deleted days ago. What else can I do to stop premature adoption?)

Reply:
Sorry for that, I've changed link to your PR.